### PR TITLE
[3.18] BPF Additional Felix config parameter for bpfForceTrackPacketsFromIfaces

### PR DESCRIPTION
**Pull request by ghost**
Additional Felix config parameter to specify interfaces allowing traffic to be tracked by Linux conntrack
NB: should only be used for interfaces that are not used for the Calico fabric
 
Product Version(s):
Calico 3.27, Calico Enterprise 3.18, and Calico Cloud 3.18

Issue:
[CORE-9468](https://tigera.atlassian.net/browse/CORE-9468)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

[CORE-9468]: https://tigera.atlassian.net/browse/CORE-9468?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: calico-cloud/reference/resources/felixconfig.mdx [hunk 1]
```diff
@@ -186,6 +186,7 @@ spec:
...
...es to skip Calico's iptables NOTRACK rule, allowing traffic from those interfaces to be tracked by Linux conntrack.  Should only be used for interfaces that are not used for the Calico fabric.  For example, a docker bridge device for non-Calico-networked containers.                                                                                                                                                                                                                                                                                                                                                                                                  | A list of strings                                                  | A list of strings                       | docker+                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (bmckercher123) [inline calico-cloud/reference/resources/felixconfig.mdx]:
```suggestion
| bpfForceTrackPacketsFromIfaces        | Forces traffic from these interfaces in BPF mode to skip Calico's iptables NOTRACK rule, allowing traffic from those interfaces to be tracked by Linux conntrack.  Use only for interfaces that are not used for the Calico fabric, for example, a docker bridge device for non-Calico-networked containers.                                                                                                                                                                                                                                                                                                                                                                                                  | A list of strings                                                  | A list of strings                       | docker+                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

**Review by bmckercher123 — APPROVED** [review]
Minor edits.

File: calico-cloud/reference/resources/felixconfig.mdx (hunk 1 above)
Comment (ghost) [reply calico-cloud/reference/resources/felixconfig.mdx]: Update description text as per suggested change
NB: updated all 3x instances for consistency

**Comment by ctauchen** [comment]
@StevenTigera Thanks for the update. Two more things:
* What versions does this apply to? Right now, you've got things in place for 3.26, 3.17, and CC/3.17. If these belong in earlier versions, you'll need to modify the corresponding file for each version.
* Needs squash + rebase, conflicts resolved, etc.

**Comment by ghost** [comment]
> @StevenTigera Thanks for the update. Two more things:
> 
> * What versions does this apply to? Right now, you've got things in place for 3.26, 3.17, and CC/3.17. If these belong in earlier versions, you'll need to modify the corresponding file for each version.
> * Needs squash + rebase, conflicts resolved, etc.

Thanks @ctauchen - squash + rebase as requested.
PS: not sure where I should update the versions therefore I have added under Product Version(s) - hope that's OK

**Comment by ctauchen** [comment]
Ok, so by "Calico 3.18, Calico Enterprise 3.18, and Calico Cloud 3.28", do you mean "OS 3.27, CE 3.18, CC/3.18"?

**Comment by bmckercher123** [comment]
@StevenTigera If this is for 3.18, we have not cut docs branches yet. And that is because we have not done 3.17.0 (early preview) or 3.17.1 (GA). From dates we have, I won't be cutting 3.18 until June.

**Comment by ghost** [comment]
> Ok, so by "Calico 3.18, Calico Enterprise 3.18, and Calico Cloud 3.28", do you mean "OS 3.27, CE 3.18, CC/3.18"?
Yes - apologies @ctauchen - I had the numbers mixed up but have corrected above - my understanding is CC aligns with OS - thanks

**Comment by ctauchen** [comment]
@StevenTigera We've finally got somewhere to put this. As it stands now, merging this PR will add these changes for OS 3.27, CE 3.18, and CC 3.18. 

Please fetch and rebase, and we can do the final checks to get this merged.

**Comment by ghost** [comment]
> @StevenTigera We've finally got somewhere to put this. As it stands now, merging this PR will add these changes for OS 3.27, CE 3.18, and CC 3.18.
> 
> Please fetch and rebase, and we can do the final checks to get this merged.
@ctauchen Done

**Comment by ctauchen** [comment]
@StevenTigera Perhaps I got the order wrong here, but I merged #644 first and now this one has conflicts. Can you resolve these?

**Comment by ghost** [comment]
> @StevenTigera Perhaps I got the order wrong here, but I merged #644 first and now this one has conflicts. Can you resolve these?

@ctauchen - NP: resolved conflicts and now two PRs 610 and 644 are in the correct order - OK to merge now - thx

**Pull request merged**